cdap-ui/app/cdap/components/Experiments/CreateView/index.js

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+import React, {Component} from 'react';
+import TopPanel from 'components/Experiments/TopPanel';
+import IconSVG from 'components/IconSVG';
+import DataPrepConnections from 'components/DataPrepConnections';
+import DataPrepHome from 'components/DataPrepHome';
+import {Prompt} from 'react-router-dom';
+import createExperimentStore from 'components/Experiments/store/createExperimentStore';
+import MyDataPrepApi from 'api/dataprep';
+import NamespaceStore from 'services/NamespaceStore';
+import UncontrolledPopover from 'components/UncontrolledComponents/Popover';
+import ExperimentPopovers from 'components/Experiments/Popovers';
+import DataPrepStore from 'components/DataPrep/store';
+import {setOutcomeColumns, setDirectives, setSrcPath, setWorkspace} from 'components/Experiments/store/ActionCreator';
+import MLAlgorithmSelection from 'components/Experiments/MLAlgorithmSelection';
+import ExperimentMetadata from 'components/Experiments/ExperimentMetadata';
+import LoadingSVGCentered from 'components/LoadingSVGCentered';
+import Helmet from 'react-helmet';
+
+require('./CreateView.scss');
+
+export default class ExperimentCreateView extends Component {
+  state = {
+    workspaceId: '',
+    isModelCreated: createExperimentStore.getState().model_create.isModelCreated,
```

Comment:
nit: don't need the comma

Reply:
FYI: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas 😉. I think we should start using this guy. Have now removed but lets discuss and we could probably have some codemod that will do this for us.

Reply:
TIL 😄